lading/src/observer/linux/procfs.rs

```diff
@@ -221,6 +223,22 @@ impl Sampler {
     ) -> Result<bool, Error> {
         let pid = process.pid();
 
+        // SAFETY: We've inserted process info into this map when polling for
+        // child processes.
+        let pinfo = self
+            .process_info
+            .get_mut(&pid)
+            .expect("catastrophic programming error");
```

Comment:
```suggestion
            .expect("pinfo catastrophic programming error");
```

just so there's an easy string to grep for should this ever be hit

Reply:
Addressed in [b6d3e8f](https://github.com/DataDog/lading/pull/1305/commits/b6d3e8f0365438e07d16f0e4e6d605f5642e245c).